docs/_content.html

```
@@ -0,0 +1,967 @@
+        <p>Or use your package manager. Arch:</p>
+        <pre><code>yay -S lazy-tmux</code></pre>
+        <p>MacOS (Homebrew):</p>
+        <pre><code>brew install alchemmist/tap/lazy-tmux</code></pre>
+        </div>
+      </details>
+
+      <details class="section">
+        <summary><h2>Quick tmux setup</h2></summary>
+        <div class="detail-body">
+        <p>Keep your tmux config clean with an include file:</p>
+        <pre><code>lazy-tmux setup >> ~/.tmux.conf</code></pre>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)